Gordon360/Gordon360.csproj

```diff
@@ -57,6 +58,9 @@
       <CopyToOutputDirectory>Always</CopyToOutputDirectory>
     </EmbeddedResource>
   </ItemGroup>
+  <ItemGroup>
```

Comment:
@EjPlatzer @jtasonye @RossClark01 Are these changes intentional?  I believe this came from a WMRA PR from a couple months ago.

Reply:
Looks like a change from a database table update that should have been reverted. We should be good to reverse this update